pkg/kubelet/util/pluginwatcher/plugin_watcher.go

```diff
@@ -91,177 +185,196 @@ func (w *Watcher) traversePluginDir(dir string) error {
 					Op:   fsnotify.Create,
 				}
 			}()
+		default:
+			glog.V(5).Infof("Ignoring file %s with mode %v", path, mode)
 		}
 
 		return nil
 	})
 }
 
-func (w *Watcher) init() error {
-	if err := w.createPluginDir(); err != nil {
-		return err
+// Handle filesystem notify event.
+func (w *Watcher) handleCreateEvent(event fsnotify.Event) error {
+	glog.V(6).Infof("Handling create event: %v", event)
```

Comment:
Is there a guideline behind using V(6) here and V(5) at L#189?

Reply:
Not really, what do you think is the right level?

Reply:
not sure either. However, lets keep same for sake of consistency.

Reply:
Noting as a refactor followup